bookkeeper-server/src/main/java/org/apache/bookkeeper/bookie/BookieShell.java

```diff
@@ -1337,6 +1354,68 @@ int runCmd(CommandLine cmdLine) throws Exception {
     }
 
     /**
+     * Setter and Getter for LostBookieRecoveryDelay value (in seconds) in Zookeeper
+     */
+    class LostBookieRecoveryDelayCmd extends MyCommand {
+        Options opts = new Options();
+
+        public LostBookieRecoveryDelayCmd() {
+            super(CMD_LOSTBOOKIERECOVERYDELAY);
+            opts.addOption("g", "get", false, "Get LostBookieRecoveryDelay value (in seconds)");
+            opts.addOption("s", "set", true, "Set LostBookieRecoveryDelay value (in seconds)");
+        }
+
+        @Override
+        Options getOptions() {
+            return opts;
+        }
+
+        @Override
+        String getDescription() {
+            return "Setter and Getter for LostBookieRecoveryDelay value (in seconds) in Zookeeper";
+        }
+
+        @Override
+        String getUsage() {
+            return "lostbookierecoverydelay [-get|-set <value>]";
+        }
+
+        @Override
+        int runCmd(CommandLine cmdLine) throws Exception {
+            boolean getter = cmdLine.hasOption("g");
+            boolean setter = cmdLine.hasOption("s");
+
+            if ((!getter && !setter) || (getter && setter)) {
+                LOG.error("One and only one of -get and -set must be specified");
+                printUsage();
+                return 1;
+            }
+            ZooKeeper zk = null;
+            try {
+                zk = ZooKeeperClient.newBuilder().connectString(bkConf.getZkServers())
```

Comment:
It would be good to put this set/get logic in BookKeeperAdmin, for two seasons:

- it keeps the shell tool simpler
- it is easier to test and verify the logic.